src/genie/libs/parser/iosxe/show_device_tracking.py

```diff
@@ -163,4 +163,122 @@ def cli(self, output=None):
         return device_tracking_database_dict
 
 
+# ======================================
+# Schema for:
+#  * 'show device-tracking database int'
+# ======================================
+class ShowDeviceTrackingDatabaseIntSchema(MetaParser):
+    """Schema for show device-tracking database int."""
+
+    schema = {
+        "binding_table": {"dynamic": int, "entries": int, "limit": int},
+        "network_layer_address": {
+            Any(): {
+                "age": str,
+                "code": str,
+                "interface": str,
+                "link_layer_address": str,
+                "prlvl": str,
+                "state": str,
+                Optional("time_left"): str,
+                "vlan": int,
+            }
+        },
+    }
+
+
+# ======================================
+# Parser for:
+#  * 'show device-tracking database int'
```

Comment:
Can you please add {interface} in command?

Reply:
Done!